.pre-commit-config.yaml

```diff
@@ -20,11 +20,11 @@ repos:
     rev: v0.730
     hooks:
      -  id: mypy
-        # We run mypy over all files because of:
-        #  * changes in type definitions may affect non-touched files.
-        #  * Running it with `mypy pandas` and the filenames will lead to
-        #    spurious duplicate module errors,
-        #    see also https://github.com/pre-commit/mirrors-mypy/issues/5
-        pass_filenames: false
         args:
-        - pandas
+          # As long as a some files are excluded from check-untyped-defs
+          # we have to exclude it from the pre-commit hook as the configuration
+          # is based on modules but the hook runs on files.
+          - --no-check-untyped-defs
```

Comment:
I would have a slight preference to remove altogether rather than using different defaults that what's in CI

Reply:
IMO it's better to do a partial `mypy` check than none at all as you're making changes.

In our PR checklist, we can ask that people do a full `mypy pandas` check before submitting.

Reply:
Given that there has been good progress in getting all untyped defs to pass, I expect this to only be a short-lived workaround.